docs/src/examples.md

```diff
@@ -511,31 +511,11 @@ julia> round(deviance(gm1), digits=5)
 In this example, we choose the best model from a set of λs, based on minimum BIC.
 
 ```jldoctest; filter = r"(\d*)\.(\d{6})\d+" => s"\1.\2"
```

Comment:
I checked out this PR locally and I am confused why the “expected” output for the subexpression `round(optimal_bic.minimum, digits=5)`  is a glm coefficients table.